extensions/azure-functions/deployment/pom.xml

```diff
@@ -82,12 +98,23 @@
         <dependency>
             <groupId>com.azure</groupId>
             <artifactId>azure-core</artifactId>
-            <version>1.33.0</version>
+            <version>1.55.3</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>io.projectreactor</groupId>
+                    <artifactId>reactor-core</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>com.azure</groupId>
-            <artifactId>azure-core-management</artifactId>
-            <version>1.8.0</version>
+            <artifactId>azure-json</artifactId>
+            <version>1.5.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-core-http-netty</artifactId>
+            <version>1.15.11</version>
```

Comment:
Should we move these versions into the BOM? (Azure-Core and Azure-Core-HTTP-Netty)